### PR TITLE
Dhoni

### PR DESCRIPTION
**Pull request by sagaryara123**

### CODE REVIEW

File: src/main/java/com/facebook/sunteam/BatMen.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * 
+ * @author sagar
+ *
+ */
+public class BatMen {
+
+	private String playerName;
+	private String playerJursyNumber;
+	private String playerRole;
+
```
Comment (livannani9) [inline src/main/java/com/facebook/sunteam/BatMen.java]: change playerName to player_name

File: src/main/java/com/sunteam/BatMen.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package com.sunteam;
+/**
+ * 
+ * @author sagar
+ *
+ */
+public class BatMen {
```
Comment (rajendraprasad20) [inline src/main/java/com/sunteam/BatMen.java:7]: class name is not BatMen, it is BatsMen,,, plz do fast

File: src/main/java/com/facebook/sunteam/BatMen.java (hunk 1 above)
Comment (livannani9) [reply src/main/java/com/facebook/sunteam/BatMen.java]: change name

File: src/main/java/com/sunteam/BatMen.java (hunk 2 above)
Comment (sagaryara123) [reply src/main/java/com/sunteam/BatMen.java]: ok...i will change

File: src/main/java/com/facebook/sunteam/BatsMen.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ * 
+ * @author sagar
+ *
+ */
+public class BatsMen {
+
+	private String playerName;
+	private String playerJursyNumber;
+	private String playerRole;
+
```
Comment (livannani9) [inline src/main/java/com/facebook/sunteam/BatsMen.java]: add one more field in string playerhight_score

File: src/main/java/com/facebook/sunteam/BatMen.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
+package com.facebook.sunteam;
+/**
+ * 
+ * @author sagar
+ *
+ */
+public class BatMen {
```
Comment (livannani9) [inline src/main/java/com/facebook/sunteam/BatMen.java]: batsmen

**Pull request merged**